MAINTAINERSHIP.md

```diff
@@ -0,0 +1,70 @@
+# Request For Maintainership
+
+This document details the acceptance process for requests from external contributors who wish for the `ibc-go` core team to take over **maintainership** of a complex block of code (ie: an application module or a light client implementation). It is the process we also follow internally for features that we develop which will go into the `ibc-go` codebase.
+
+For projects who have created a custom IBC application and want us to list this application on the registry, please break out your module into its own repo for ease of import into `ibc-go`!
+
+- Create a repo with the custom module in a folder `x/` or `modules/`. 
+- Please include an app that contains the custom module along with end-to-end tests that spins up the blockchain and runs the custom module. 
+- See [cosmos/interchain-security](https://github.com/cosmos/interchain-security) for an example of this setup.
+
+For contributors wishing to submit contributions to the codebase, please check our [Contributor Guidelines](https://github.com/cosmos/ibc-go/blob/main/CONTRIBUTING.md) :)
+
+
+<p align="center">
+  <img src="maintainership.png?raw=true" alt="maintainership" width="80%" />
+</p>
+
+## Step 1: Product check
+
+Reach out to the IBC product team through susannah@interchain.io to coordinate use-case walkthrough.
+
+Answer these questions in a requirements doc:
```

Comment:
In addition, there should be a need for ibc-go maintainers to take over this code. We should encourage the community not to be reliant on us